### PR TITLE
Use JavaScript localeCompare to sort "Name" field in dataTable.

### PR DESCRIPTION
**Pull request by dzhuang**
It helps in sorting user full names which are not using English characters (e.g., Chinese), and some languages which won't provide correct ordering with alphabetical sorting (e.g., Spanish).

### CODE REVIEW

**Comment by inducer** [comment]
`participation-table.html` gets used from the "participation query" function, too--does this break that?

**Comment by dzhuang** [comment]
I think not. That seems to only change how the tables are rendered, without affecting the query results.

File: course/templates/course/gradebook.html [hunk 1]
```diff
@@ -50,7 +50,8 @@
         "scrollX": true,
         "scrollCollapse": true,
         "paging": true,
-        "ordering": false,
```
Comment (dzhuang) [inline course/templates/course/gradebook.html]: I have no idea why ``"ordering"`` is originally set to be ``false``. It works well if set to be ``true``.

**Comment by inducer** [comment]
Looks good. Thanks!

**Comment by dzhuang** [comment]
I think I figure out why you originally disable sorting in the gradebook-- the sorting of grade opportunity won't give expected result?

**Pull request merged**